addons/mail/data/mail_templates_email_layouts.xml

```diff
@@ -222,6 +236,27 @@
 </table>
 </td></tr>
 </table>
+</body></html>
+        </template>
+
+        <template id="preview_with_padding">
+<div style="display: none; max-height: 0px; overflow: hidden;">
+    <t t-if="tracking_values">
+        <t t-esc="tracking_values[0][0]"/>: <t t-esc="tracking_values[0][1]"/> -&gt; <t t-esc="tracking_values[0][2]"/>
+        <t t-if="len(tracking_values) > 1"> |...</t>
+        <t t-if="message.hidden_preview"> | </t>
+    </t>
+    <t t-out="message.hidden_preview"/>
+<!-- Whitespace to push back email content so that it doesn't appear in preview. Specific characters to use are susceptible to change over time -->
+&#847; &#847; &#847; &#847; &#847; &#847; &#847; &#847; &#847; &#847; &#847; &#847; &#847; &#847; &#847; &#847; &#847; &#847; &#847; &#847;
```

Comment:
can you do something like this ?
```xml
<t t-out"'&#847;' * 100"/>
```

Reply:
I think it's better if you alternate white-space with zero width unicode `&#8203;`
I read on a blog that some mail client trim the text to build the preview (so this character break the trim)